src/besselj.jl

```diff
@@ -220,3 +220,16 @@ function log_besselj_small_arguments_orders(v, x::T) where T
     logout = -loggamma(v + 1) + v * log(x/2) + log(out)
     return logout
 end
+
+# valid when x < v (uniform asymptotic expansions)
+function besselj_debye(v, x)
+    T = eltype(x)
+    S = promote_type(T, Float64)
+    x = S(x)
+    n = v*log(v/x + sqrt((v/x)^2 - 1)) - sqrt(v^2 - x^2)
+    coef = inv(sqrt(2*T(pi))) * exp(-n) / (v^2 - x^2)^(1/4)
```

Comment:
`(v^2 - x^2)^(1/4)` will be much slower than `sqrt(sqrt(v^2 - x^2))` (especially since you already are using `sqrt(v^2 - x^2)`)

Reply:
Fixed now. I need to go through next and see if we are losing any precision anywhere which I have not done. Just getting these to 'work' for now in BigFloat.